contracts/liquidators/LiquidationAuction02.sol

```diff
@@ -129,4 +161,12 @@ contract LiquidationAuction02 is ReentrancyGuard {
             collateralToBuyer = collateralInPosition;
         }
     }
+
+    function isCurveLP(address asset) public view returns(bool) {
+        address underlying = wrappedToUnderlyingOracle.assetToUnderlying(asset);
```

Comment:
Why are we unwrapping? Aren't Curve LP tokens allowed as-is?

Reply:
The [wrapper contract](https://github.com/iamdefinitelyahuman/curve-dao-contracts/blob/feat-gauge-wrapper-usdp/contracts/gauges/LiquidityGaugeWrapperUnit.vy) is used to allow users to receive gauge rewards providing collateral to the Unit Protocol